avaframe/ana1Tests/simiSolTest.py

```diff
@@ -0,0 +1,828 @@
+"""
```

Comment:
I strongly suggest to rename the one letter functions like h u v. Especially since u v h etc are also used as variable names!